pkgs/by-name/ls/lsp-ai/package.nix

```diff
@@ -0,0 +1,87 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  darwin,
+  pkg-config,
+  cmake,
+  openssl,
+  zlib,
+  perl,
+  gitUpdater,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lsp-ai";
+  version = "0.7.1";
+  src = fetchFromGitHub {
+    owner = "SilasMarvin";
+    repo = "lsp-ai";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mBbaJn4u+Wlu/Y4G4a6YUBWnmN143INAGm0opiAjnIk=";
+  };
+
+  passthru.updateScript = gitUpdater {
```

Comment:
passthru should be right before meta

Reply:
More of a general question: Why aren't these semantic requirements built into the linter/formatter? Seems like a lot of DevOps/Review cycles being wasted for things that could be automated or checked initially?

Reply:
@ProjectInitiative there are many derivations that have slightly different conventions, and nixpkgs is just very big, so writing linter/formatter that works on nixpkgs scale just takes time and effort. We could start small at just enforce some rough order or well known attributes, someone just need to volunteer their time